hedera-node/hedera-app/src/main/java/com/hedera/node/app/HederaInjectionComponent.java

```diff
@@ -153,6 +153,9 @@ interface Builder {
         @BindsInstance
         Builder systemFileUpdateFacility(SystemFileUpdateFacility systemFileUpdateFacility);
 
+        @BindsInstance
+        Builder exchangeRateManager(ExchangeRateManager exchangeRateManager);
```

Comment:
Please also try to compile xtest (after applying my latest PR). Unfortunately, it is not built automatically yet. It defines a custom dagger configuration which is often broken after changes like this.

Reply:
It looks like it's working. When I applied your changes the tests are compiling and running but they are failing(similar to https://github.com/hashgraph/hedera-services/pull/8181).